samples/samples/snippets.py

```diff
@@ -1860,6 +1936,34 @@ def query_data_with_numeric_parameter(instance_id, database_id):
     # [END spanner_query_with_numeric_parameter]
 
 
+def query_data_with_json_parameter(instance_id, database_id):
+    """Queries sample data using SQL with a JSON parameter. """
+    # [START spanner_query_with_json_parameter]
+    # instance_id = "your-spanner-instance"
+    # database_id = "your-spanner-db-id"
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+    database = instance.database(database_id)
+
+    example_json = json.dumps({"rating": 9})
+    param = {"details": example_json}
+    param_type = {"details": param_types.JSON}
+
+    with database.snapshot() as snapshot:
+        results = snapshot.execute_sql(
+            "SELECT VenueId, VenueDetails\n"
+            + "FROM Venues\n"
```

Comment:
nit: `+` is unneeded; Python will append these without it. See `query_data_with_array` sample for example.